### PR TITLE
Fix slugification of app name/alias used for the git remote name

### PR DESCRIPTION
**Pull request by hsablonniere**
See #166 for the details.

### CODE REVIEW

File: src/models/app_configuration.js [hunk 1]
```diff
@@ -42,7 +41,7 @@ AppConfiguration.addLinkedApplication = function(appData, alias) {
     app_id: appData.id,
     deploy_url: appData.deployment.httpUrl || appData.deployment.url,
     name: appData.name,
-    alias: alias || unidecode(appData.name).replace(/[^a-zA-z0-9]+/gi, "-").toLowerCase()
+    alias: alias || appData.name
```
Comment (divarvel) [inline src/models/app_configuration.js]: can you slugify the app name in that case as well?

Comment (hsablonniere) [reply src/models/app_configuration.js]: I don't get what you mean.

The slugification of the alias happens directly in `Git.createRemote()`. When do you want to slufigy the name and what would it imply ?

Comment (divarvel) [reply src/models/app_configuration.js]: The alias is also used with the `--alias` flag. If we automatically generate an alias for the user (if they don't provide it), we make it console-friendly

Comment (divarvel) [reply src/models/app_configuration.js]: If the user specifies an alias when creating / linking the app, then we use it (we assume the user knows what they're doing)

Comment (hsablonniere) [reply src/models/app_configuration.js]: OK, so if no alias is provided through CLI, we should slugify the name and store the slugified version as an alias.

I think this means we should move the `slugify()` out of `Git.createRemote()` and back in `app_configuration.js`, WDYT?

Comment (divarvel) [reply src/models/app_configuration.js]: we still need to have it called in `Git.createRemote`, as users may pass invalid git remote names (anyway `slugify` is idempotent so it's not an issue)

Comment (divarvel) [reply src/models/app_configuration.js]: if the user wants to have a specific alias it's not an issue for CLI arguments, but for git remotes it's an issue

**Comment by divarvel** [comment]
👌 thanks!

**Pull request merged**